tests/routes/test_withings_routes.py

```diff
@@ -93,7 +93,7 @@ async def test_first_user_weight(
         userid=withings_user.oauth_userid,
         startdate=1683894606,
         enddate=1686570821,
-        db=mocked_session,
+        db=mocked_async_session,
```

Comment:
See if we can use an http client in tests, instead of calling the endpoints directly: 

https://fastapi.tiangolo.com/advanced/testing-database/